### DISCUSSION TITLE
Provide installation instructions without using docker

### DISCUSSION THREAD

**Discussion by Kunsi** [immich-app/immich#2520] [Feature Request]
### The feature

The [installation instructions](https://immich.app/docs/install/requirements) are basically:
- use the installation script (which uses docker-compose)
- use docker-compose
- use kubernetes (which uses docker containers)
- use portainer (which uses docker containers)
- use unraid (which uses docker-compose)

Please add instructions on how to install Immich without the use of docker containers.

This software looks really great, but (for reasons i cannot explain here) i cannot use docker in my environment. Putting it into a VM or a LXC container would work, though.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1657. Note that we can't provide support for installations like this.
